lib/generators/rys/plugin/plugin_generator.rb

```diff
@@ -115,6 +115,7 @@ def create_patches
 
     def create_test_files
       directory 'spec'
+      directory 'test'
```

Comment:
To ne. Tam má být jen dummy aplikace.

Reply:
no a kam tu dummy mám dát, když na ní chybí adresář ? 🙂